### PR TITLE
Add DataStax.com logo

### PR DESCRIPTION
**Pull request by eolivelli**
### Motivation
DataStax recently acquired Kesque.com, this patch replaces the references to Kesque.com with DataStax.

### Modifications
Update the website

### CODE REVIEW

**Comment by eolivelli** [comment]
@Jennifer88huang please take a look

**Review by codelipenghui — APPROVED** [review]

File: site2/website/data/users.js [hunk 1]
```diff
@@ -287,10 +287,10 @@ module.exports = [
...
     },
     {
-        name: 'Kesque',
-        url: 'https://kesque.com',
-        logo: 'https://static.kafkaesque.io/wp-content/uploads/2020/06/kesque_logo.svg',
-        logo_white: true
+        name: 'DataStax',
+        url: 'https://datastax.com',
+        logo: 'https://www.datastax.com/sites/default/files/inline-images/datastax-logotype-negative.png',
+        logo_white: false
```
Comment (Jennifer88huang) [inline site2/website/data/users.js:293]: I notice the logo is transparent(white), you need to set it to `true` (adding black background), which would be sth like this:
image

Or you can set it to "false" or not set it if you use this link for your logo: https://www.datastax.com/_next/static/images/logo-6e2e31572c645818cd3d2ac070d67778.svg

**Review by Jennifer88huang — COMMENTED** [review]
@eolivelli Thank you very much for adding DataStax logo on Pulsar website.

File: site2/website/data/users.js (hunk 1 above)
Comment (Jennifer88huang) [reply site2/website/data/users.js:293]: You can [build locally](https://github.com/apache/pulsar/tree/master/site2#run-the-site-locally) to check the display effect on Pulsar website.

Comment (eolivelli) [reply site2/website/data/users.js:293]: @Jennifer88huang thanks for your suggestion.

This is how the site looks like to me.
I have played with log_white a bit and this looks to me the best option.

Screenshot 2021-02-05 at 16 26 52

I also have the logo with black background and transparent characters, but I was not able to make it work properly
https://www.datastax.com/sites/default/files/inline-images/datastax-logotype-positive.png

**Comment by eolivelli** [comment]
@Jennifer88huang I have set logo_write to `true`
please take a look

**Review by merlimat — APPROVED** [review]

**Comment by Jennifer88huang** [comment]
@eolivelli Good job~

**Pull request merged**